tools/download_go

```diff
@@ -84,14 +84,34 @@ fi
 #
 # One word to describe this process might be "brittle".
 #
-if ! name=$(/usr/bin/awk -v "v=$GOVERSION" -v "o=$GOOS" -v "a=$GOARCH" -F\" '
-    BEGIN { pattern = "^go"v"."o"-"a".tar.bz2$"; }
-    $1 ~ pattern { print $1 }' <<< "$list") ||
-    [[ -z "$name" ]]; then
-	printf 'ERROR: could not locate file name in index page\n' >&2
-	printf '\t(Does Go version %s (%s-%s) exist?)\n' \
-	    "$GOVERSION" "$GOOS" "$GOARCH" >&2
-	exit 1
+if [[ "$GOOS" = 'solarisish' ]]; then
```

Comment:
Did you intentionally want to avoid the more common "solarish" term? I suppose a more self-explanatory name might be "illumosElseSolaris", but not a biggie.

Reply:
Yep, I was avoiding that in case it looked like a typo (though I've heard the term before, others reading the script might miss it) illumosElseSolaris is even better though. I'll switch to that - we're not in a rush here, so let's do it properly.